client/extensions/woocommerce/app/store-stats/referrers/index.js

```diff
@@ -124,8 +133,26 @@ class Referrers extends Component {
 					query={ query }
 					statType={ STAT_TYPE }
 					title={ title }
-					queryParams={ queryParams }
+					queryParams={ periodNavQueryParams }
 				/>
+				<Module
+					className="referrers__chart"
+					siteId={ siteId }
+					emptyMessage={ translate( 'No data found' ) }
```

Comment:
Perhaps we can add a single "no data found"/"no activity"/"No referral activity on this date" etc. constant to client/extensions/woocommerce/app/store-stats/constants.js to ensure the string is used consistently?